src/main/java/org/broadinstitute/hellbender/tools/funcotator/DataSourceFuncotationFactory.java

```diff
@@ -234,5 +228,6 @@ private boolean isFeatureListCompatible(final List<Feature> featureList) {
     /**
      * @return Get the {@link Class} of the feature type that can be used to create annotations by this {@link DataSourceFuncotationFactory}.
      */
-    protected abstract Class<? extends Feature> getAnnotationFeatureClass();
+    @VisibleForTesting
```

Comment:
The `@VisibleForTesting` annotation means that the method is "more widely visible than otherwise necessary, only for use in test code.", which can be applicable for public methods.

Reply:
I believe it is, since we are expanding the accessibility only for testing.

Reply:
No action

Reply:
Ah.  I misunderstood.  I had not seen this annotation used for this in the past.